src/modules/fancyzones/editor/FancyZonesEditor/ViewModels/MonitorViewModel.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Windows;
+using FancyZonesEditor.Utils;
+
+namespace FancyZonesEditor.ViewModels
+{
+    public class MonitorViewModel : INotifyPropertyChanged
+    {
+        private const int MaxPreviewDisplaySize = 160;
+        private const int MinPreviewDisplaySize = 98;
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public delegate void MonitorChangedEventHandler(MonitorChangedEventArgs args);
+
+        public ObservableCollection<MonitorInfoModel> Monitors { get; set; }
+
+        public static double DesktopPreviewMultiplier { get; private set; }
+
+        public Visibility DesktopsPanelVisibility
+        {
+            get
+            {
+                return App.Overlay.MultiMonitorMode ? Visibility.Visible : Visibility.Collapsed;
+            }
+        }
+
+        public RelayCommand AddCommand { get; set; }
+
+        public RelayCommand DeleteCommand { get; set; }
+
+        public RelayCommand<MonitorInfoModel> SelectCommand { get; set; }
+
+        public MonitorViewModel()
+        {
+            SelectCommand = new RelayCommand<MonitorInfoModel>(SelectCommandExecute, SelectCommandCanExecute);
+
+            Monitors = new ObservableCollection<MonitorInfoModel>();
+            double maxDimension = 0, minDimension = double.MaxValue;
+
+            int i = 1;
+            foreach (var monitor in App.Overlay.Monitors)
+            {
+                Device device = monitor.Device;
+                var bounds = device.UnscaledBounds;
```

Comment:
We should use the `ScaledBounds` since we want to show the actual dimensions not the native monitor resolution.